### PR TITLE
bitcore -> bitcore-lib

### PR DESCRIPTION
**Pull request by braydonf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ A Bitcoin full node for building applications and services with Node.js. A node
 ## Install
```
Comment (kleetus) [inline README.md]: should the title of this README be 'Bitcore' instead of 'Bitcore Node'?

Comment (braydonf) [inline README.md]: I don't think so. We can handle that on the Bitcore README.

**Comment by kleetus** [comment]
I am not really sure how to test this branch. Specifically, how does the bitcore, bitcored runnables get installed?

File: bin/bitcore-node [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env node
+// vi: ft=javascript -*- mode: JavaScript; -*-
```
Comment (kleetus) [inline bin/bitcore-node]: Please remove this line. This has nothing to do with bitcore-node

**Pull request merged**